### PR TITLE
Add missing tests

### PR DESCRIPTION
**Pull request by smalyala**
- Add tests for modify file client
- Add testing for `input` arg in `list_datum`
- Add tests for license mixin (in `test_enterprise.py`)
- Add tests for identity mixin

### CODE REVIEW

**Review by bbonenfant — APPROVED** [review]
LGTM! Although I think it would be a step in the right direction to move towards validating the contents of the cluster itself rather than the amount of data contained within the cluster.

File: tests/test_identity.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+@util.skip_if_no_enterprise()
+def test_oidc_client(client):
+    oidc1 = client.create_oidc_client(
+        identity_proto.OIDCClient(id="oidc1", name="pach1", secret="secret1")
+    )
+    client.create_oidc_client(
+        identity_proto.OIDCClient(id="oidc2", name="pach2", secret="secret2")
+    )
+
+    assert len(client.list_oidc_clients()) == 2
```
[the remark this answers is not in the document]
Comment (smalyala) [reply tests/test_identity.py:59]: Just to make sure I'm understanding correctly, you're asking if we can go a step further than counting the number and actually see if the OIDC Clients in `client.list_oidc_clients()` match the ones we created?

[the remark this answers is not in the document]
Comment (bbonenfant) [reply tests/test_identity.py:59]: Sorry I missed this notification, but yes is that possible to do?

[the remark this answers is not in the document]
Comment (smalyala) [reply tests/test_identity.py:59]: We have a get method `get_oidc_client()` to inspect with OIDC clients are created. I can add in a check after `list_oidc_clients().

**Pull request merged**